src/job.ts

```diff
@@ -3,22 +3,26 @@ import {NewJobCommittedMessage} from './committed-message'
 import {Nullable} from './nullable'
 
 const NO_JOB = '--no-job--'
+export const NO_JOB_ID = -1
```

Comment:
@yeraydavidrodriguez I think we do not need to make it public if we use the `nullJobId` function when we create the `JobId` class.